resources/definitions/creality_ender3.def.json

```
@@ -57,7 +57,7 @@
         },
         "layer_height_0": {
             "default_value": 0.2
-        },
+        },        
```

remove trailing spaces
```suggestion
        },
```

---

Thanks! Must have missed that.